compiler/compile/OMRCompilation.cpp

```diff
@@ -2713,3 +2714,17 @@ OMR::Compilation::CompilationPhaseScope::~CompilationPhaseScope()
    {
    _comp->restoreCompilationPhase(_savedPhase);
    }
+
+TR::Region &OMR::Compilation::aliasRegion()
+   {
+   return self()->_aliasRegion;
+   }
+
+void OMR::Compilation::invalidateAliasRegion()
+   {
+   if (self()->_aliasRegion.bytesAllocated() > (0.75) * TR::Region::initialSize())
```

Comment:
The 0.75 was not scientifically tuned - it was chosen to avoid recreating the region after every opt, but to try and make sure that we had sufficient space to allocate at least a few BitVectors before the region would need to allocate from the allocator. I will pull the 0.75 out into a `#define`